apps/client/src/pages/DashboardList.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+import Table from '@cloudscape-design/components/table';
+import Box from '@cloudscape-design/components/box';
+import Button from '@cloudscape-design/components/button';
+import TextFilter from '@cloudscape-design/components/text-filter';
+import Header from '@cloudscape-design/components/header';
+import Pagination from '@cloudscape-design/components/pagination';
+import CollectionPreferences from '@cloudscape-design/components/collection-preferences';
+import { Link } from 'react-router-dom';
+
+const DashboardList: React.FC = () => {
```

Comment:
Can you please add unit tests to for this table component?

Reply:
Agreed we'll want tests for this once this component actually handles real data. This is already tested with the route tests though. I feel unit testing this component at the moment would be redundant.

Reply:
The page is hit in the tests, but there's no functionality of the table tested. I will be inclined to agree to not test yet if you remove the implementations for pagination, filtering, collection preferences, loading state, column visibility, selection state, and aria labels, leaving it a simple list of dashboards.

Reply:
Agreed. Let me simplify this since we don't need this functionality yet. Copied from Cloudscape.

Reply:
Thank you!